src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,11 +1,25 @@
-export const TodoFilter = () => (
+/* eslint-disable jsx-a11y/control-has-associated-label */
+interface Props {
```

Comment:
Generally there is a different way of typing setters for state:
![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/87986955/541c331a-74b0-4a56-9a5c-844bd6fa5b97)

Reply:
And below typing of the select:
![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/87986955/ae069fc3-3b45-456a-930c-078daa6de56c)